### DISCUSSION TITLE
cant connect High Sierra to UGS

### DISCUSSION THREAD

**Discussion by shuffleshoes** [winder/Universal-G-Code-Sender#2339] [Help] [answered]
Hello, I am struggling to connect UGS with my XYX carve. Theres no com port available. I'm running MacOs High Sierra on a macbook pro. I have succeed before using a mac running Catalina, but I bought myself an old macbook to use as a dedicated CNC machine, its older so it runs on High Sierra 10.13.1 now I cant connect. I have the latest nightly build of UGS 2.0.21

**Comment by breiler** [comment, marked as answer]
What controller are you using?

Are you sure that you've installed the proper driver for it?

**Reply by shuffleshoes** [reply]
The controller is GRBL-AIO - I've installed FTDI VCP Driver and now it works, thanks for that :)
